app.test.js

```diff
@@ -94,4 +94,34 @@ describe("GET /api/articles/:article_id", () => {
             expect(body.message).toBe("invalid id path")
         })
     })
+})
+
+
+describe("GET /api/articles", () => {
+    test("returns all articles on this endpoint", () => {
+        return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({body}) => {
+            let resBody = body.response
+
+            expect(Object.keys(resBody[2])).toHaveLength(8)
+            expect(resBody)
+
+            expect(resBody[7]).toEqual(
```

Comment:
Rather than picking a random article to check - it would be more thorough to loop through all the articles to check they all have the shape you're expecting